src/vm/costs/cost_functions.rs

```diff
@@ -130,4 +130,5 @@ define_named_enum!(ClarityCostFunction {
     NftOwner("cost_nft_owner"),
     NftBurn("cost_nft_burn"),
     PoisonMicroblock("poison_microblock"),
+    StxGetAccount("cost_stx_get_account"),
```

Comment:
I didn't see where this cost function is defined?  Is this waiting on a hypothetical `.costs-2` contract that will add it in, as well as changes to the runtime cost-tracker to know which `.costs-X` contract to use?

Reply:
Removed -- this needs to be added when addressing #2717.